src/toolbars/TimeToolBar.cpp

```diff
@@ -264,7 +264,7 @@ void TimeToolBar::SetResizingLimits()
 // Called when the project settings change
 void TimeToolBar::OnSettingsChanged(wxCommandEvent &evt)
 {
-   evt.Skip(false);
+   evt.Skip();
```

Comment:
This one line deserves a tiny commit of its own because it is a bug fix in existing code, not properly part of your project.